src/genai/__init__.py

```diff
@@ -6,8 +6,6 @@
 from genai.client import Client
 from genai.credentials import Credentials
 
-logging.getLogger(__name__).addHandler(logging.NullHandler())
```

Comment:
removing `NullLogger` as discussed, because we want to show deprecation warnings even if the user hasn't configured logging